tests/unit_tests/test_wave_03.py

```diff
@@ -131,7 +131,11 @@ def test_swap_items_from_their_empty_returns_false():
 
     result = fatimah.swap_items(jolie, item_b, nobodys_item)
 
-    raise Exception("Complete this test according to comments below.")
+    assert len(fatimah.inventory) == 3
+    assert len(jolie.inventory) == 0
+    assert not result
```

Comment:
These look good! One small thing is that earlier in a different test, you assert `result == False`. While both of these approaches work and do essentially the same thing, try your best to keep things consistent across the testing suite! Even between testing files!